pkg/agent/openflow/pipeline.go

```diff
@@ -1781,6 +1773,26 @@ func (c *client) serviceEndpointGroup(groupID binding.GroupIDType, withSessionAf
 	return group
 }
 
+// decTTLFlows decrements TTL by one for the packets from a local Pod to a remote Pod. Antrea doesn't
+// decrement TTL for the packets which enter OVS pipeline from antrea-gw0 on the source Node.
+// TTL is also decremented on the destination Node in L3Routing Table when the packet is received.
+// See the functions of "l3FlowsToPod", "l3ToPodFlow", and "l3ToGWFlow".
+func (c *client) decTTLFlows(category cookie.Category) []binding.Flow {
+	var flows []binding.Flow
+	for _, proto := range c.ipProtocols {
+		flows = append(flows, c.pipeline[l3DecTTLTable].BuildFlow(priorityNormal).MatchPriority(priorityNormal).
+			MatchProtocol(proto).
+			MatchRegRange(int(marksReg), markTrafficFromLocal, binding.Range{0, 15}).
+			Action().DecTTL().
+			// Bypass l2ForwardingCalcTable and tables for ingress rules (which won't
+			// apply to packets to remote Nodes).
+			Action().GotoTable(conntrackCommitTable).
+			Done(),
```

Comment:
```suggestion
			Done()
```

Reply:
Since the ")" for function "append" is in another line, here we need a ",",  otherwise there would be syntax error. If I remove the ending "," here, I should move the ")" to this line.